framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.16 under development
 ------------------------
 
+- Bug #16926: Fix shell autocompletion (GHopperMSK)
```

Comment:
This actually does not make sense - completion script is not distributed with framework, so it should not be mentioned in changelog.

Reply:
@samdark Where? I cant find it at https://github.com/yiisoft/yii2-framework

Reply:
Just look at extracted commit from this PR: https://github.com/yiisoft/yii2-framework/commit/b1d822809acee9c3c64fe844de3fbcf8b4c185e1 - it is just entry in changelog, without affecting framework itself.

Reply:
Ah, subsplit. Yes, you are right.

Reply:
https://github.com/yiisoft/yii2/commit/a4b12b14cacb0d9ddac3b71015aaa047d1580b9a